frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -138,6 +138,17 @@ class Worksheet extends React.Component {
                         info,
                     },
                 });
+                if (info.blocks) {
+                    for (let block of info.blocks) {
+                        if (
+                            block.sort_keys &&
+                            Object.values(block.sort_keys).some((sort_key) => sort_key == null)
```

Comment:
Add a comment mentioning this is a temporary solution, and put a link to a github issue that describes this issue (make the github issue if needed).

Great that you're able to fix it! Is there a chance this might cause an infinite loop though?

Reply:
Isn't this really inefficient because you have to save the entire worksheet for each block?  At least you could just save it once?  But this is definitely not satisfying because we really want to only save the relevant block, not the whole worksheet.  This is really important for long worksheets, and this will cause a huge performance degradation.  Ideally, we'd get to the root cause and fix that.

Reply:
@epicfaace @percyliang 
I think the root cause of this issue is:
If a user runs something like cl run "echo 2", this will be added to a worksheet without sort_key. This messes up the frontend logic to determine where to insert bundle.

I think the fix should be 
1. From the backend, add it to the beginning of a worksheet if it's added from CLI without -a.
2. for existing worksheet, we keep my current fix so they will get re-parsed and the sort key gets fixed.

Reply:
1. Wait, `cl` commands should always add bundles to the end of the worksheet, not the beginning.
2. If we just make sure everything gets a sort key when bundles are added from the backend, then why do we need to do anything on the frontend?

Reply:
2. It's because existing worksheet will still have bundles will null sort key. The re-save will just happen once when they open those worksheets for the first time so those nulls get fixed. Or we can manually fix existing ones.

Reply:
I see - it's just a one time thing.  I think it's cleanest to just do a migration to fix the nulls.  Or if not that, then doing it on the backend seems like it'd make more sense (i.e., whenever we fetch a worksheet, if there are nulls, we fix it on the spot).

Reply:
> then doing it on the backend seems like it'd make more sense (i.e., whenever we fetch a worksheet, if there are nulls, we fix it on the spot).

I like this approach.